src/travel-details-screens/components/Trip.tsx

```diff
@@ -185,6 +187,9 @@ export const Trip: React.FC<TripProps> = ({
                           vehicleWithPosition: legVehiclePosition,
                           mode: leg.mode,
                           subMode: leg.transportSubmode,
+                          onScreenClose: async () => {
+                            await requestReview();
+                          },
```

Comment:
Could we remove the await here, and make it fire-and-forget? (Same for departure details)

```suggestion
                          onScreenClose: () => {
                            requestReview();
                          },
```